index.src.html

```diff
@@ -1242,6 +1260,7 @@ <h2 id="sample-reports">Sample Reports</h2>
         "type": "csp",
         "age": 10,
         "url": "https://example.com/vulnerable-page/",
+        "user_agent": "ReportingSpec/1",
```

Comment:
This is a bit confusing and easy to misinterpret as version of reporting API. Perhaps use a real-world UA string as an example?

Reply:
#98